tests/include/detray/test/utils/ratio_test.hpp

```diff
@@ -0,0 +1,130 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
Still I don't like the name of `ratio_test.hpp`
This file doesn't contain any thing about ratio and it can be confused with `ratio` class in `core/utils` directoy

Reply:
I calculated the ratio between the number of points that pass vs those that don't pass. I am open to suggestions on how to call it

Reply:
Do you even have to define `ratio_test` function? Now I look at the function and I don't understand why you need to define super general function to count the number of points that do not pass the predicate.

I also don't understand what `(Number of points - Number of points that do not pass) / Number of points that do not pass` means

Reply:
Isn't it better to switch the denominator with Number of points?

Reply:
```
(Number of points - Number of points that do not pass) = Number of points that pass
```
so this returns 
```
Number of points that pass / Number of points that do not pass
```
Either one could be zero, so why is it better?

Reply:
Probably I am not following your code well.

I understood that you are calculating `(success/failure)` in ratio test.

I thought it would be better to calculate `(success/(success+failure))`

Please let me know if we are on the same page

Reply:
Oh sure, I can do that, too